jupyter-book/conditions/differential_gene_expression_environment.yml

```diff
@@ -0,0 +1,20 @@
+name: deg
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python=3.8
+  - scanpy=1.8.2
+  - pandas=1.2.5
```

Comment:
Somewhat outdated given that the latest is 1.5.2. Think that this shouldn't be an issue if we upgrade it?

Reply:
will check, should be alright with the latest too